rdr_service/etl/bq/queries.py

```diff
@@ -0,0 +1,951 @@
+   src_c.topic_value                   AS topic_value,
+   src_c.value_code_id                 AS value_code_id,
+   COALESCE(vc3.concept_id, 0)         AS value_source_concept_id,
+   CASE
+      WHEN src_c.is_invalid = 1 THEN 2000000010
+      ELSE COALESCE(vc4.concept_id, 0)
+   END                                 AS value_concept_id,
+   src_c.value_number                  AS value_number,
+   src_c.value_boolean                 AS value_boolean,
+   CASE
+      WHEN src_c.value_boolean = 1 THEN 45877994
+      WHEN src_c.value_boolean = 0 THEN 45878245
+      ELSE 0
+   END                                 AS value_boolean_concept_id,
+   src_c.value_date                    AS value_date,
+   src_c.value_string                  AS value_string,
+   src_c.questionnaire_response_id     AS questionnaire_response_id,
+   src_c.unit_id                       AS unit_id,
+   src_c.is_invalid                    as is_invalid,
+   src_c.src_id                        AS src_id
+FROM `{dataset_id}.src_clean` src_c
+JOIN `{dataset_id}.src_participant` src_p
+   ON  src_c.participant_id = src_p.participant_id
+LEFT JOIN `{dataset_id}.concept` vc1
+   ON  src_c.question_ppi_code = vc1.concept_code
+   AND vc1.vocabulary_id = 'PPI'
+LEFT JOIN `{dataset_id}.concept_relationship` vcr1
+   ON  vc1.concept_id = vcr1.concept_id_1
+   AND vcr1.relationship_id = 'Maps to'
+   AND vcr1.invalid_reason IS NULL
+LEFT JOIN `{dataset_id}.concept` vc2
+   ON  vcr1.concept_id_2 = vc2.concept_id
+   AND vc2.standard_concept = 'S'
+   AND vc2.invalid_reason IS NULL
+LEFT JOIN `{dataset_id}.concept` vc3
+   ON  src_c.value_ppi_code = vc3.concept_code
+   AND vc3.vocabulary_id = 'PPI'
+LEFT JOIN `{dataset_id}.concept_relationship` vcr2
+   ON  vc3.concept_id = vcr2.concept_id_1
+   AND vcr2.relationship_id = 'Maps to value'
+   AND vcr2.invalid_reason IS NULL
+LEFT JOIN `{dataset_id}.concept` vc4
+   ON  vcr2.concept_id_2 = vc4.concept_id
+   AND vc4.standard_concept = 'S'
+   AND vc4.invalid_reason IS NULL
+WHERE src_c.filter = 0
+       """, 'destination': 'src_mapped', 'append': False},
+    'src_meas_mapped': {'query': """
+        SELECT
+  meas.participant_id AS participant_id,
+  meas.finalized_site_id AS finalized_site_id,
+  meas.code_value AS code_value,
+  COALESCE(tmp1.cv_source_concept_id, 0) AS cv_source_concept_id,
+  COALESCE(tmp1.cv_concept_id, 0) AS cv_concept_id,
+  tmp1.cv_domain_id AS cv_domain_id,
+  meas.measurement_time AS measurement_time,
+  meas.value_decimal AS value_decimal,
+  meas.value_unit AS value_unit,
+  COALESCE(vc1.concept_id, 0) AS vu_concept_id,
+  meas.value_code_value AS value_code_value,
+  COALESCE(tmp2.vcv_source_concept_id, 0) AS vcv_source_concept_id,
+  COALESCE(tmp2.vcv_concept_id, 0) AS vcv_concept_id,
+  meas.measurement_id AS measurement_id,
+  meas.physical_measurements_id AS physical_measurements_id,
+  meas.parent_id AS parent_id,
+  meas.src_id AS src_id,
+  meas.collect_type AS collect_type
+FROM
```

Comment:
Can the indentation for the queries be updated? Ideally everything would be indented to match with the indentation level of where it is in the dictionary. But if not that, then maybe decrease the indentation of the SELECT so that it lines up with the FROM.